orion/sites/admin/src/programs/programEdit.js

```diff
@@ -178,9 +178,47 @@ export class ProgramEditPage extends React.Component {
 }
 
 function checkProgram(program) {
-  // *todo* change this later!
```

Comment:
I think this is here because you haven't pulled from master after that today's  merge yet.